docs/integration.md

```diff
@@ -131,4 +147,3 @@ m.mount(document.getElementById("widget2-container"), Widget2)
 
 m.endComputation()
 ```
-
```

Comment:
I'm guessing this was your editor misbehaving (i.e. not honoring the [EditorConfig](http://editorconfig.org/) here)?